### PR TITLE
skip Ember run loop creation in test mode.

### PR DESCRIPTION
**Pull request by iezer**
@jamesarosen Simple fix to disable the run-loop in test mode only and get our tests passing. Once we merge this master I'll try to get it merged to the main ride/ember-stripe-service repo.

### CODE REVIEW

**Comment by iezer** [comment]
@jamesarosen  Modified this to reject with `Ember.Error`

File: app/services/stripe.js [hunk 1]
```diff
@@ -37,11 +37,10 @@ function createCardToken (card) {
       debug('card.createToken handler - status %s, response:', status, response);
 
       if (response.error) {
-        return Ember.run(null, reject, response);
+        Ember.run(null, reject, new Ember.Error(response.code));
```
Comment (jamesarosen) [inline app/services/stripe.js]: You removed the `return`, so this will try to both `reject` and `resolve`.

**Comment by iezer** [comment]
@jamesarosen thank you I put back the return statement.

**Comment by jamesarosen** [comment]
+1

**Pull request merged**